modules/navier_stokes/test/tests/postprocessors/tests

```diff
@@ -26,6 +26,15 @@
     cli_args = "Mesh/inactive='diverging_mesh' Outputs/file_base='straight_INSFV' -pc_type lu -pc_factor_shift_type NONZERO"
     ad_indexing_type = 'global'
   []
+  [fv_straight_different_interpolation]
+    type = CSVDiff
+    input = conservation_INSFV.i
+    csvdiff =  straight_INSFV_different_interpolation.csv
+    abs_zero = 1e-8
+    requirement = 'The system shall be able to compute flow rates and, when using a different interpolation method between kernels and postprocessors, show approximate mass, momentum and energy conservation at internal and external boundaries of a frictionless heated straight channel with a finite volume incompressible Navier Stokes model.'
+    cli_args = "Postprocessors/mid1_momentum_x/advected_interp_method='upwind' Postprocessors/mid2_momentum_x/advected_interp_method='upwind' Postprocessors/mid1_advected_energy/advected_interp_method='upwind' Postprocessors/mid2_advected_energy/advected_interp_method='upwind' Mesh/inactive='diverging_mesh' Outputs/file_base='straight_INSFV_different_interpolation' -pc_type lu -pc_factor_shift_type NONZERO"
+    ad_indexing_type = 'global'
+  []
```

Comment:
Is this something we actually care to test? E.g. is this something we would actually want our users to do?

Reply:
Well we dont want users to do that, but we also dont want that to fail hard like it was when the bug was there.

Reply:
Well ideally I think I would like it to "fail hard" when the postprocessor interpolation method is different than the kernels :smile: E.g. it would be great if it actually `mooseErrored`

Reply:
oh yeah mooseErrored would have been fine!
I can delete the test if you dont like the requirement. 
This test covers the same code as other tests, it just covers one more potential bug.

Reply:
Ok I'm not actually sure what bug you're referring to...

Reply:
In the fixup to the FV postproc PR, when face UO were not computing the material properties

Reply:
You mean the framework code I added? Material properties were being computed. The problem was that the variable values used in the neighbor material property computation were garbage. I am honestly unsure as to why only one test from your postprocessor test quite showed issues. I would have expected all tests with the internal volumetric flow rate postprocessor to be affected by the previous neighbor variable value garbage.

Reply:
I thought the case you showed me with serial vs parallel was for material properties though. You were printing _advected_quantity right? (the weird 347 in serial)

My guess as to why only one case was wrong was that when the same scheme was used, the kernels, executed before the face UO, were causing the values to be computed. 

But when a different interpolation method was used, then values that were not initialized got queried by the Face UO, hence the garbage.

But it really is just a guess. Ok looks like we're looking at deletion for this test then.

Reply:
> I thought the case you showed me with serial vs parallel was for material properties though. You were printing _advected_quantity right? (the weird 347 in serial)

I was printing the temperature variable value from the material.

> My guess as to why only one case was wrong was that when the same scheme was used, the kernels, executed before the face UO, were causing the values to be computed.

This is a reasonable guess in theory. However, we compute user objects in an entirely different element loop than kernels. so what I would expect is that a user object without properly initialized variable data would, when computing on the first element/face, be seeing variable data from the *last* computed element/face from the kernel loop, or whatever element loop went immediately before the user object computation. And I would've expected this kind of bad behavior everywhere we were using that internal volumetric flow object. But seemingly not!

> But it really is just a guess. Ok looks like we're looking at deletion for this test then.

Yea I'm hoping that we're getting coverage on proper initialization of variable data from our different mixes of serial, parallel, and threaded testing with the "proper" way of setting `advected_interp_method` for both kernels and postprocessors. The fact that we had the one failure that we fixed with the proper variable initialization gives me some confidence that we do indeed have coverage without the mixed `advected_interp_method` test.